### DISCUSSION TITLE
[Question][Hw4] Campaign.env format change

### DISCUSSION THREAD

**Discussion by lemonadeeeee** [prosyslab-classroom/cs348-information-security#555] [Q&A]
I was wondering if we can safely change the format of Campaign.env. Specifically, I was wondering if it is safe to add new data to env, in addition, to the ones that already exist. (#step, #pass, etc)

**Comment by KihongHeo** [comment]
It is up to you. If you want, you can add anything. Make sure not to break the meta data part.
